src/components/designer/taro/info/AssetsList.tsx

```diff
@@ -10,6 +11,9 @@ import AssetInfoDrawer from './AssetInfoDrawer';
 
 const Styled = {
   Wrapper: styled.div``,
+  P: styled.p`
```

Comment:
This should have a more descriptive name, such as `Empty`.

Reply:
Done.